bolt-service/scripts/setup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+BOLT_SERVER_PACKAGE=pe-bolt-server_0.22.0.7.g3257f59-1bionic_amd64.deb
+
+# Install Puppet-Agent and Bolt Server
+apt update -y
+apt install -y curl vim
```

Comment:
What does this need vim for? Is vi not there already?

Reply:
Yeah we can take out vim